app/models/legislation.rb

```diff
@@ -130,7 +109,7 @@ def events_with_eventable_title
   end
 
   def date_passed
-    events.where(event_type: 'law_passed').first&.date
+    events.where(event_type: 'passed/approved').first&.date
```

Comment:
why not just `passed.first&.date`

Reply:
I don't know :D I just changed the event type :P but good catch, I'll update it =)